app/views/comments/_comment.html.haml

```diff
@@ -1,9 +1,4 @@
 - if in_list
-  %li
-    - if with_link_wrapper
-      = link_to application_path(comment.application), title: "View application at #{comment.application.address}", class: "panel-link" do
-        = render 'comments/comment_item', comment: comment, with_address: true
-    - else
-      = render 'comments/comment_item', comment: comment, with_address: false
+  %li= render 'comments/comment_item', comment: comment, with_address: (with_address ? true : false)
```

Comment:
Does this need this ternary true/false statement if `with_address` already evaluates true or false?